.github/workflows/run-server-tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Run Server Tests
+
+run-name: Running Server Tests for ${{ github.ref }}
+
+on: [push]
+
+env:
+  PG_USER_TEST: brave
+  PG_DATABASE_TEST: brave
+  PG_PASSWORD_TEST: actions
+  PG_PORT_TEST: 5432
+  PG_HOST_TEST: localhost
```

Comment:
Maybe try `postgres` instead of `localhost`. (Ref: https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers#running-jobs-in-containers ).